src/test/integration/integration-test.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import json
+import unittest
+import yaml
+import utils
+
+
+class integration_tests(unittest.TestCase):
+    @classmethod
+    def setup(cls, config_path, openapi_path):
+        with open(config_path) as config_file:
+            config = json.load(config_file)
+            cls.base_url = utils.setup_base_url(config)
+            cls.session = utils.setup_session(config)
+            cls.test_cases = config['test_cases']
```

Comment:
configuration-example does not have any field for 'test_cases'. Though we don't want to commit specific usernames, we still want these fields present with fake example values. For example, valid_authority_onids could have values like "replace_with_valid_onid". A note of this should be made in the README as part of the setup instructions.